### ISSUE TITLE
Ads work on only some devices in production.

### ISSUE THREAD

**Issue by DoruLaz** [admob-plus/admob-plus#64]
### Is this a bug report?

Somewhat.

### Environment

AdMob Plus Environment Info:
    System:
      OS: Windows 7
      CPU: (4) x64 Intel(R) Core(TM) i3-3250 CPU @ 3.50GHz
      Memory: 6.12 GB / 15.96 GB
    Binaries:
      Node: 8.11.2 - C:\Program Files\nodejs\node.EXE
      npm: 2.15.12 - D:\Projects\OQ\omniquiz\node_modules\.bin\npm.CMD
    IDEs:
      Android Studio: Version  2.2.0.0 AI-145.3537739

cordova: 6.4.0
ionic: 2.2.1

cordova plugins: {
  "cordova-admob-plus": "~0.24.1"
}

AndroidManifest.xml: looks ok

### Actual Behavior

Hi,

I am successfully using this plugin in a production app, for both Android and iOS.
The problem is that my error reporter says that for some devices the ads won't load because of **isTrusted=false**. This is not the case where the error code is 3 (No fill) because I check for that before sending the error.

NOTE: I have read the other 2 bugs that involve **isTrusted=false**, and yes I am building my app in release and also it is signed (since I managed to upload it to the store). The strange thing is that on the majority of devices it works, but only on some I get this error. And always on Android, so far none have been for iOS.

Any ideas on this?

**Comment by ratson** [comment]
if it works on one device, it is unlikely the problem of this library, see https://admob-plus.github.io/docs/faq.html#why-my-ads-is-not-showing

**Comment by DoruLaz** [comment]
Yes, I understand. But the thing is that it is not the case with No Fill error. That one has error code 3 which I am handling. But from yesterday I had reports of at least 100 times the ad did not load, and the error code is not 3. All I get is that **isTrusted=false**. This is an old ad unit id, in production built in release...
